src/tableDesigner/tableDesignerWebviewController.ts

```diff
@@ -169,9 +169,7 @@ export class TableDesignerWebviewController extends ReactWebviewPanelController<
                     server: connectionInfo.server,
                     database: databaseName,
                     connectionString: connectionString,
-                    accessToken: connectionInfo.azureAccountToken
-                        ? connectionInfo.azureAccountToken
-                        : undefined,
+                    accessToken: connectionInfo.azureAccountToken,
```

Comment:
If the account isn't an azure account token then this will be undefined and it will work like it did previously?